exes/mqtt-broadcaster/inc/mqtt_broadcaster.hpp

```diff
@@ -377,6 +459,46 @@ class mqtt_broadcaster {
     }
   }
 
+  auto send_value_on_signal(std::string signal_name,
+                            std::variant<bool, double, std::string, int64_t, uint64_t, uint32_t> value) -> void {
+    for (auto& sig : scada_signals) {
+      if (sig.definition.name == signal_name) {
```

Comment:
could we skip storing the definition, and only store the signal? We could expose the name in the public API of IPC signal and do a comparision of signal_name.ends_with(signal.name()) within the std visit clause

Reply:
Done, please see changes in ipc layer.